swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -2120,9 +2127,8 @@ paths:
         - $ref: "#/components/parameters/dateFrom"
         - $ref: "#/components/parameters/dateTo"
         - $ref: "#/components/parameters/entryReferenceFrom"
-        - $ref: "#/components/parameters/bookingStatus"
+        - $ref: "#/components/parameters/bookingStatusCard"
```

Comment:
Relevant Change: bookingStatusCardrenamed bookingStatus
Adoption approach: Follow the change